OpenRA.Mods.Cnc/Traits/Infiltration/Infiltrates.cs

```diff
@@ -119,7 +119,7 @@ public void ResolveOrder(Actor self, Order order)
 			if (!order.Queued)
 				self.CancelActivity();
 
-			self.SetTargetLine(target, Color.Red);
+			self.ShowTargetLines();
```

Comment:
Can we do this from the same place that we show the target/move flash?  That seems like it would be the most natural place to me.